src/app/main.scss

```diff
@@ -224,6 +224,7 @@ input[type='search'] {
   padding-right: constant(safe-area-inset-right);
   padding-right: env(safe-area-inset-right);
   scroll-margin-top: $header-height;
+  margin-bottom: 101px;
```

Comment:
This'll probably cause scrolling to go weird in the app in general - we'll probably need a more surgical approach.